Sources/StreamVideo/Controllers/CallController.swift

```diff
@@ -317,7 +342,13 @@ class CallController {
             if reconnectionDate != nil {
                 reconnectionDate = nil
             }
-            call?.update(reconnectionStatus: .connected)
+            executeOnMain { [weak self] in
+                guard let self else { return }
+                let status = self.call?.state.reconnectionStatus
+                if status != .migrating {
+                    self.call?.update(reconnectionStatus: .connected)
```

Comment:
what if status is `.disconnected` or `.reconnecting`? Do we still want to overwrite the it?

Reply:
yes, since the case is `.connected`. This is not enough for migrating only.